lib/web_console/config/locales/en.yml

```diff
@@ -0,0 +1,15 @@
+en:
```

Comment:
Maybe drop the config dir and live the locales there. We can even try to follow Rails' conventions and put it in top-level config dir. Even more if our Railtie was an Engine, we could have get an initializer to load the locales (in `config/locales`) for free.

However, I would like to avoid the overhead of subclassing Engine, just for using one of its the many initializers, so I think `lib/web_console/locales` can work fine for us. :+1: